LEAF_Request_Portal/templates/reports/LEAF_Inbox.tpl

```diff
@@ -189,6 +189,12 @@
         },
     };
 
+    function scrubHTML(input) {
```

Comment:
A concern I have with this.  It doesn't do what it says it's doing.  It's just wrapping it in a div.  I would recommend either a different name or fully explain what it's _meant_ to do in comments inside the function itself.